src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -499,6 +509,10 @@ public void run() {
 			if (this.count > 5) {
 				this.cancel();
 			} else if (this.count == 1) {
+				if(ThreadLocalRandom.current().nextDouble() < .1){

```

Comment:
There should be a space between the if and opening parenthesis. Additionally, there should be a space between the closing parenthesis and the opening brace.

```java
if (ThreadLocalRandom.current().nextDouble() < .1) {
```